pkg/apply/desiredset.go

```diff
@@ -225,6 +226,13 @@ func (o desiredSet) WithNoDelete() Apply {
 	return o
 }
 
+func (o desiredSet) WithNoDeleteGVK(gvk ...schema.GroupVersionKind) Apply {
+	for _, curr := range gvk {
+		o.noDeleteGVK[curr] = struct{}{}
```

Comment:
I believe you have to initialize the map first.

Reply:
Still drinking coffee over here, thank you.